packages/components/src/Autocomplete/Autocomplete.tsx

```diff
@@ -69,20 +69,23 @@ interface AutocompleteProps
 
 // Max statements increased to make room for the debounce functions
 /* eslint max-statements: ["error", 14] */
-export function Autocomplete({
-  initialOptions = [],
-  value,
-  allowFreeForm = true,
-  size = undefined,
-  debounce: debounceRate = 300,
-  onChange,
-  getOptions,
-  placeholder,
-  onBlur,
-  onFocus,
-  validations,
-  ...inputProps
-}: AutocompleteProps) {
+function AutocompleteInternal(
```

Comment:
hmmm, it should work. I believe @jessedoyle leverages that

Reply:
🤔 we should reach out and talk to him. I am surprised it worked because it "should" be a forward ref from my understanding of refs from the [docs](https://react.dev/reference/react/forwardRef#forwarding-a-ref-through-multiple-components)

Reply:
In my tests it didn't work at anytime without adding the `forwardRef` as per @MichaelParadis suggestion!

Reply:
I'd be shocked if this works without `forwardRef` unless I've really misunderstood the docs.

Reply:
Ever noticed how you don't even have to declare `ref` as a prop. It just happens the moment you `forwardRef`. It'll work without it, the downside is that you have to manually expose it

> Ref forwarding is a technique for automatically passing a [ref](https://legacy.reactjs.org/docs/refs-and-the-dom.html) through a component to one of its children. This is typically not necessary for most components in the application

[source](https://legacy.reactjs.org/docs/forwarding-refs.html#:~:text=Ref%20forwarding%20is%20a%20technique%20for%20automatically%20passing%20a%20ref%20through%20a%20component%20to%20one%20of%20its%20children.%20This%20is%20typically%20not%20necessary%20for%20most%20components%20in%20the%20application)

Now if they do change something in v19, that'll get fucked haha so we need to expose the element on the ref as well instead of a limited version of it.

Reply:
Ah okay, that does make sense. 

In terms of leaving this/removing it, maybe we should leave it for now but deprecate that approach and get folks to just use the forwarded ref?

Reply:
That is a good compromise @mattander

Reply:
leave it as is, if this PR passes review then we merge it and we'll mark that other thing deprecated. Otherwise, if there are more changes, we could add that deprecation task here

Reply:
Ok, now I'm confused 😆 should we remove the `inputRef` then? @MichaelParadis @darryltec are you ok with that? Then I can update the PR for your approval and we can move on 🥳

Reply:
Based on @jessedoyle 's comment I think deprecation and removing in the future makes sense in case others are using it

Reply:
Yup! Going into more detail about it. Deprecation and removal isn't really a straightforward thing as we do need to replace that functionality if we are removing it.